test/test-proxied-properties.js

```diff
@@ -178,6 +189,12 @@ describe("browser-polyfill", () => {
 
         window.chrome.runtime.onMessage.addListener(() => {});
         equal(lazyInitCount, 1, "chrome.runtime should be re-used upon accessing chrome.runtime");
+
+        window.browser.privacy.network.networkPredictionEnabled();
```

Comment:
Don't call the function, just check that the returned object looks like a Setting object (object with set, get and clear methods).

Reply:
Done.